netlab_testing/HTTPTests.cpp

```diff
@@ -51,193 +60,38 @@ class HTTP_Tests : public test_base {
 		http_client = new HTTPClient_Impl();
 	}
 
-	void set_HTTP_flavor(HTTPProtocol http_protocol) {
-		http_server->set_HTTP_procotol(http_protocol, inet_server);
-		http_client->set_HTTP_procotol(http_protocol, inet_client);
-	}
+	void TearDown() override {
```

Comment:
implemented in test base

Reply:
Might need to delete HTTP client and server objects, we'll see.